src/templates/app/cashier/payment_methods.jsx

```diff
@@ -413,9 +414,8 @@ const PaymentMethods = () => {
                                 ],
                             },
                             {
-                                id      : 'jeton',
-                                dataShow: '-eucountry',
-                                row     : [
+                                id : 'jeton',
```

Comment:
there is another file where jeton has the flag -eucountry, can you remove it from there too?

Reply:
Sure